util/cron/test-valgrindexe.bash

```diff
@@ -2,16 +2,14 @@
 #
 # Test valgrind-compatible configuration on full suite with valgrind on linux64.
 
-# Use LLVM-13 to work around https://github.com/Cray/chapel-private/issues/3373
-# and pretend it's a system llvm to avoid common.bash adding a newer one
-source /cray/css/users/chapelu/setup_system_llvm.bash 13
-export OFFICIAL_SYSTEM_LLVM=true
-
 CWD=$(cd $(dirname $0) ; pwd)
 source $CWD/common.bash
 source $CWD/common-valgrind.bash
 source $CWD/common-localnode-paratest.bash
 
+# Use LLVM-13 to work around https://github.com/Cray/chapel-private/issues/3373
+source /cray/css/users/chapelu/setup_system_llvm.bash 13
```

Comment:
It is not really relevant to this PR, but FYI I have found a workaround for this problem that allows running valgrind with LLVM 15. It is to do something along these lines and then rebuild the runtime:

```
export CHPL_TARGET_CC='clang-15 -gdwarf-4'
export CHPL_TARGET_CXX='clang++-15 -gdwarf-4'
export CHPL_TARGET_LD='clang++-15 -gdwarf-4'
``

It's also my understanding that sufficiently new valgrinds will support Dwarf 5.

Reply:
Oh good. I will leave that to a future PR (and I think there may be more instances that were using an old version to work around? not sure)